src/Time/Units.hs

```diff
@@ -132,6 +135,27 @@ type AllTimes =
    , Millisecond , Microsecond, Nanosecond, Picosecond
    ]
 
+{- | Creates the list of time units in descending order by provided
+the highest and the lowest bound of the desired list.
+
+__Usage example:__
+
+>>> seriesF @(Hour ... Second) $ hour 3 +: minute 5 +:sec 3 +: ms 123
+"3h5m3s"
+
+-}
+type family (...) (a :: Rat) (b :: Rat) where
```

Comment:
I think it's better to give more meaningful names to types and variables. Like `from` and `to`.